src/main/java/roomescape/domain/ReservationStatus.java

```diff
@@ -0,0 +1,26 @@
+package roomescape.domain;
+
+public enum ReservationStatus {
+
+	RESERVATION("예약"), WAITING("예약 대기");
+
+	private final String status;
+
+	ReservationStatus(String status) {
+		this.status = status;
+	}
+
+	public String getStatus() {
+		return this.status;
+	}
+
+	public static String fromStatus(String savedValue) {
+		for (ReservationStatus status : values()) {
+			if (status.name().equals(savedValue)) {
+				return status.getStatus();
+			}
+		}
+		throw new IllegalArgumentException("Unknown value: " + savedValue);
+	}
```

Comment:
Stream API 를 사용할 수도 있겠네요.

```suggestion
		return Arrays.stream(values())
				.filter(status -> status.name().equals(savedValue))
				.map(ReservationStatus::getStatus)
				.findFirst()
				.orElseThrow(() -> new IllegalArgumentException("Unknown value: " + savedValue));
```

Reply:
done! https://github.com/next-step/spring-roomescape-waiting/pull/21/commits/51693d4ba73e1494a0194c31f7dad7c403240ad7